app/models/ext_management_system.rb

```diff
@@ -128,6 +128,9 @@ def hostname_uniqueness_valid?
   virtual_column :total_vms_suspended,     :type => :integer
   virtual_total  :total_subnets,           :cloud_subnets
 
+  virtual_total  :connection_definitions, :type => :array
```

Comment:
Does this work? Looking at the definition for `virtual_total` it looks like it should take a `name` and a `relation` argument.